### PR TITLE
implements database with fake data seeded

### PR DESCRIPTION
**Pull request by shabbyjoon**
faker.js file contains db and fake data generator

### CODE REVIEW

**Comment by shabbyjoon** [comment]
Please review my code @wissemgamra

File: faker.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+ for (let j = 0; j < results.length; j++){
+     connection.query(`INSERT INTO artist (artist_name, listeners, artist_image) VALUES("${results[j].artist_name}", "${results[j].listeners}", "${results[j].artist_image}")`, function(error, result, fields) {
+         if (error) {
+             console.log(error);
+         }
+         
+     })
+ }
+
```
Comment (wissemgamra) [inline faker.js]: Code looks good! Indentation is a little off. I believe it should be 2 spaces instead of 4.

Comment (wissemgamra) [inline faker.js]: Code looks good! Indentation is a little off. I believe it should be 2 spaces instead of 4.

File: package.json [hunk 2]
```diff
@@ -8,10 +8,23 @@
...
+    "body-parser": "^1.18.3",
+    "express": "^4.16.3",
+    "faker": "^4.1.0",
+    "mysql": "^2.16.0",
+    "nodemon": "^1.18.4",
+    "react": "^16.5.0",
+    "react-dom": "^16.5.0"
   },
   "devDependencies": {
-    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor"
```
Comment (wissemgamra) [inline package.json]: I believe that we should leave the eslint so that the code looks good!

File: schema.sql [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+-- DROP TABLE IF EXISTS relatedArtists;
+		
+CREATE TABLE relatedArtists (
+   id INT AUTO_INCREMENT PRIMARY KEY,
+   related_Artist_ID INT NOT NULL,
+   main_Artist_ID INT NOT NULL,
+   FOREIGN KEY (related_Artist_ID) REFERENCES artist(artistID),
+   FOREIGN KEY (main_Artist_ID) REFERENCES artist(artistID)
+ );
+
```
Comment (wissemgamra) [inline schema.sql]: Looks great!

**Pull request merged**